EmarsysInbox/EmarsysInboxController.swift

```diff
@@ -102,18 +102,18 @@ extension EmarsysInboxController: UITableViewDataSource, UITableViewDelegate {
         cell.titleLabel.text = message.title
         cell.bodyLabel.text = message.body
         
-        guard let imageUrl = message.imageUrl, let url = URL(string: imageUrl) else {
-            cell.messageImageView.image = EmarsysInboxConfig.defaultImage
+        guard let iconUrl = message.properties?["icon"], let url = URL(string: iconUrl) else {
```

Comment:
we should keep `message.imageUrl` as a fallback, to avoid changing the current behaviour, and `properties?["icon"]` is optional / not well known when creating inbox campaign